romancal/assign_wcs/pointing.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+from astropy.modeling.models import RotationSequence3D, Scale
+from gwcs.geometry import SphericalToCartesian, CartesianToSpherical
+
+
+def v23tosky(input_model, wrap_v2_at=180, wrap_lon_at=360):
+    """ Create the transform from telescope to sky.
+
+    The transform is defined with a reference point in a Frame
+    associated tih the telescope (V2, V3) in arcsec, the corresponding
+    reference poiont on sky (RA_REF, DEC_REF) i deg, and the position angle
+    at the center of the aperture, ROLL_REF in deg.
+
+    Parameters
+    ----------
+    input_model : `roman_datamodels.WfiImage`
+        Roman imaging exposure data model.
+    wrap_v2_at : float
+        At what angle to wrap V2. [deg]
+    wrap_lon_at : float
+        At what angle to wrap logitude. [deg]
```

Comment:
Could include `return' section in epydoc for m.

Reply:
done